lib/resourcemerge/core_test.go

```diff
@@ -34,8 +119,16 @@ func TestEnsurePodSpec(t *testing.T) {
 			}
 
 			if !equality.Semantic.DeepEqual(test.existing, test.expected) {
-				t.Errorf("mismatch PodSpec got: %v want: %v", test.existing, test.expected)
+				t.Errorf("mismatch PodSpec got:\n%s\nwant:\n%s", yamlOrDie(test.existing), yamlOrDie(test.expected))
```

Comment:
You can use https://godoc.org/k8s.io/apimachinery/pkg/util/diff

Reply:
+1 to using the k8s lib (which we already vendor).